packages/core/content-manager/server/controllers/validation/model-configuration.js

```diff
@@ -32,7 +43,10 @@ const createSettingsSchema = (schema) => {
       // should be reset when the type changes
       mainField: yup.string().oneOf(validAttributes.concat('id')).default('id'),
       // should be reset when the type changes
-      defaultSortBy: yup.string().oneOf(validAttributes.concat('id')).default('id'),
+      defaultSortBy: yup
+        .string()
+        .oneOf(validAttributes.concat(['id', ...nonVisibleWritableAttributes]))
```

Comment:
I was wondering should we allow sorting by created by & updated by also (maybe the other PR should do that 🤔 )

Reply:
updated to allow also creator fields [`f6543f7` (#17205)](https://github.com/strapi/strapi/pull/17205/commits/f6543f7003bdeef8bffd518a9ddfbd8f96e300ae)